src/full-snap-points.ts

```diff
@@ -119,7 +114,7 @@ export const createFullSnapshotForNetwork = async (network: Network) => {
     (acc, curr) => {
       if (curr.name === InvariantEventNames.CreatePositionEvent) {
         const event = parseEvent(curr) as CreatePositionEvent;
-        if (!isPromotedPool(PROMOTED_POOLS, event.pool)) return acc;
+        if (event.currentTimestamp.lt(START_COUNT_TIMESTAMP)) return acc;
```

Comment:
You shouldn't need start ocunt timestamp, the first tranasction from mainnet should be enough. Even it is better to refer to transaction hashes than to timestamps